### PR TITLE
Pro 1234 demo merge for bsp tests

### PR DESCRIPTION
**Pull request by sanjayparekh-hmcts**
Merge to master branch for testing BSP on demo env

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/probate/controller/ExceptionRecordController.java [hunk 1]
```diff
@@ -92,14 +96,17 @@
             switch (formType) {
                 case PA8A:
                     callbackResponse = erService.createCaveatCaseFromExceptionRecord(erRequest, warnings);
+                    logTransformationCallback(callbackResponse);
```
Comment (davejones74) [inline src/main/java/uk/gov/hmcts/probate/controller/ExceptionRecordController.java]: Are these logs to be disabled now or after UAT?

File: src/main/java/uk/gov/hmcts/probate/service/NotificationService.java [hunk 2]
```diff
@@ -353,7 +355,7 @@ private SendEmailResponse getSendEmailResponse(State state, String templateId, S
                                                    String reference)
         throws NotificationClientException {
         SendEmailResponse response;
-
+        log.info("getSendEmailResponse");
```
Comment (davejones74) [inline src/main/java/uk/gov/hmcts/probate/service/NotificationService.java]: Debug comment?

File: src/main/java/uk/gov/hmcts/probate/service/NotificationService.java [hunk 3]
```diff
@@ -362,8 +364,11 @@ private SendEmailResponse getSendEmailResponse(State state, String templateId, S
             case CASE_STOPPED_REQUEST_INFORMATION:
             case REDECLARATION_SOT:
             default:
+                log.info("send the email to notificationClient");
                 response = notificationClient.sendEmail(templateId, emailAddress, personalisation, reference);
+                log.info("sent the email to the client!");
         }
+        log.info("Return the SendEmailResponse: {} " , response );
```
Comment (davejones74) [inline src/main/java/uk/gov/hmcts/probate/service/NotificationService.java:368]: More debug comments?

**Review by davejones74 — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/probate/controller/ExceptionRecordController.java (hunk 1 above)
Comment (sanjayparekh-hmcts) [reply src/main/java/uk/gov/hmcts/probate/controller/ExceptionRecordController.java]: I did remove them, but decided to take them from demo - ill remove them again. They should not be in master anyway

File: src/main/java/uk/gov/hmcts/probate/service/NotificationService.java (hunk 2 above)
Comment (sanjayparekh-hmcts) [reply src/main/java/uk/gov/hmcts/probate/service/NotificationService.java]: taken from demo code - I can remove

File: src/main/java/uk/gov/hmcts/probate/service/NotificationService.java (hunk 3 above)
Comment (sanjayparekh-hmcts) [reply src/main/java/uk/gov/hmcts/probate/service/NotificationService.java:368]: taken from demo code - I can remove

**Review by davejones74 — APPROVED** [review]

**Pull request merged**